### DISCUSSION TITLE
Alanwar TV 2

### DISCUSSION THREAD

**Discussion by Mennaruuk** [iptv-org/community#702] [Q&A]
**Channel Name:** Al-Anwar 2
**Channel ID:** AnwarTV2.iq
**Language:** Arabic
**Website:** https://anwartv2.com/
**Stream URL (optional):** https://player.castr.com/live_c043ce9040aa11ec94bb9191b5291a5e
**Notes (optional):** They have a livestream, ~~but it's tokenized.~~

**Comment by Mennaruuk** [comment]
Stream url: https://player.castr.com/live_c043ce9040aa11ec94bb9191b5291a5e

**Reply by sabah1973** [reply]
http://cloud.odysee.live/content/f92670235a1ce2bce4cf77671cc4dcc2188baa1d/master.m3u8

**Comment by BellezaEmporium** [comment]
I think I can work with this.

**Comment by Alstruit** [comment]
https://dacastmmd.mmdlive.lldns.net/dacastmmd/573db53ae58b429c8ec4d1d9563ab60f/playlist.m3u8

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/iq.m3u#L58-L62
